tests/integration/test_agentd/test_agentd_reconnection.py

```diff
@@ -220,7 +219,7 @@ def test_agentd_reconection_enrollment_with_keys(configure_authd_server, start_a
     # Wait until Agent is notifying Manager
     try:
         log_monitor.start(timeout=120, callback=wait_notify)
-    except TimeoutError as err:
+    except TimeoutError:
         remoted_server.stop()
         raise AssertionError("Notify message from agent was never sent!")
```

Comment:
We can remove this "conversion" from `TimeoutError` to `AssertionError` by just adding the error message to the monitor.